src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -150,6 +150,14 @@ using namespace mongo;
 
 /* ****************************************************************************
 *
+* OtisTreatFunction - callback signature for treatOntimeintervalSubscriptions
+*/
+typedef void (*OtisTreatFunction)(std::string tenant, BSONObj& bobj);
```

Comment:
How many functions of this class we have in the code? Appart of the one for storing DB csubs into the cache I haven't seen any other (but the PR is long and maybe I'm missing something).